front/components/actions/tables_query/TablesQueryActionDetails.tsx

```diff
@@ -153,18 +154,33 @@ function QueryTablesResults({
       <span className="text-sm font-semibold text-foreground dark:text-foreground-night">
         Results
       </span>
-      <div>
-        <Citation
-          className="w-48 min-w-48 max-w-48"
-          containerClassName="my-2"
-          onClick={handleDownload}
-          tooltip={title}
-        >
-          <CitationIcons>
-            <Icon visual={TableIcon} />
-          </CitationIcons>
-          <CitationTitle>{title}</CitationTitle>
-        </Citation>
+      <div className="flex flex-row gap-2">
```

Comment:
I think it could be convenient to offer getting the results both in csv and json so I would keep both. WDYT?

Reply:
I still don't see the value of the JSONL here? Not a common format and it does not bring new information does it?